mlflow/projects.py

```diff
@@ -349,45 +354,52 @@ def _fetch_git_repo(uri, version, dst_dir):
         repo.heads.master.checkout()
 
 
+def _get_conda_env_name(conda_env_path):
```

Comment:
Split conda-env-creation activation functionality out into helper functions for testing